sdk/src/test/java/io/dapr/client/DaprClientHttpTest.java

```diff
@@ -107,79 +107,54 @@ public void setUp() {
 
   @Test
   public void waitForSidecarTimeOutHealthCheck() throws Exception {
```

Comment:
The root cause was global changes performed by these test cases in DaprClientHttpTest and it affected RetryPolicyTest run if it happens after these.